client/my-sites/checkout/checkout-thank-you/style.scss

```diff
@@ -833,6 +833,71 @@
 	}
 }
 
+main.akismet-checkout-thank-you {
+	max-width: 1000px;
+
+	.akismet-checkout-thank-you__main-message,
+	.akismet-checkout-thank-you__email-message,
+	.akismet-checkout-thank-you__email-message-loading {
+		margin: 1em 0;
+		letter-spacing: -0.02em;
+
+		@include break-mobile {
+			max-width: 450px;
+		}
+	}
+
+	.akismet-checkout-thank-you__email-message {
+		font-size: $font-title-small;
+	}
+
+	.akismet-checkout-thank-you__main-message {
+		font-weight: 400;
+		font-size: $font-headline-medium;
+		line-height: 1.2;
+		margin-top: 0;
+
+		@include break-mobile {
+			margin-top: 1.5em;
+		}
+	}
+
+	.akismet-checkout-thank-you__emoji {
+		font-size: 40px; /* stylelint-disable-line declaration-property-unit-allowed-list */
```

Comment:
Explanation for using an ignore comment: Designer wanted this to be 40px, and there is no way with the allowed `rem` values or with the existing variables that I am aware of to get this value without hard coding. I tried using 2.5rem but I got a lint error saying to use one of the allowed values, and there was 2.25 and 2.75 but nothing in between.